site/content/en/docs/tasks/run_python_jobs.md

```diff
@@ -310,7 +310,7 @@ kubectl get pods -n kueue-system
 When Kueue is ready:
 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kueue/main/site/static/examples/single-clusterqueue-setup.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kueue/main/site/static/examples/admin/single-clusterqueue-setup.yaml
```

Comment:
```suggestion
kubectl apply -f github.com/kubernetes-sigs/kueue/site/static/examples/admin/single-clusterqueue-setup.yaml
```

I think this should work, but please test

Reply:
It does not work, i have test it

```
kubectl apply -f github.com/kubernetes-sigs/kueue/site/static/examples/single-clusterqueue-setup.yaml
error: the path "github.com/kubernetes-sigs/kueue/site/static/examples/single-clusterqueue-setup.yaml" does not exist
```